frontend/packages/console-shared/src/utils/storage-utils.ts

```diff
@@ -12,3 +13,9 @@ export const isCephProvisioner = (scProvisioner: string): boolean => {
     _.endsWith(scProvisioner, provisioner),
   );
 };
+
+// To check that the storage class isn't noobaa based
+export const filterScOnProvisioner = (provisioner: string = '', show: boolean = true) =>
```

Comment:
we could simplify this to 
```suggestion
export const filterScOnProvisioner = (sc: StorageClass, provisioner: string): boolean => sc.provisioner.includes(provisioner);
```
and use `useCallback` later

also the comment above does not apply anymore as you can specify any storage class, not just noobaa